base/file.jl

```diff
@@ -280,6 +281,18 @@ function rm(path::AbstractString; force::Bool=false, recursive::Bool=false)
     end
 end
 
+"""
+    rmrf(path::AbstractString)
+
+Forcibly and recursively remove the provided path.
+Equivalent to `rm(path, force=true, recursive=true)`.
```

Comment:
```suggestion
Shorthand for `rm(path, force=true, recursive=true)`.
```

Reply:
My reasoning being that we often say "equivalent to" when we mean "does effectively the same thing but in a different way" whereas here it's literally the same as that.